packages/node_modules/samples/browser-call-with-screenshare/index.html

```diff
@@ -44,16 +44,46 @@
       <main>
         <h1>Local Screensharing</h1>
         <p>This sample demonstrates sharing your local screen</p>
+
+        <form id="typeCredentials">
+          <fieldset>
+            <legend>Pick Method of Getting Access Token</legend>
+              <div>
+                <input type="radio" id="oauth" name="drone" onchange="handleRadio()" value="oauth" checked>
+                <label for="huey">OAuth</label>
+              </div>
+              <div>
+                <input type="radio" id="paste" name="drone" onchange="handleRadio()" value="paste">
+                <label for="huey">Paste Access Token</label>
```

Comment:
```suggestion
                <input type="radio" id="oauthLogin" name="tokenType" onchange="handleRadio()" value="oauth" checked>
                <label for="oauthLogin">OAuth</label>
              </div>
              <div>
                <input type="radio" id="tokenLogin" name="tokenType" onchange="handleRadio()" value="paste">
                <label for="tokenLogin">Paste Access Token</label>
```

Updating the `label` to match the id will allow the user to click the label to select the radio element.